packages/webview_flutter/webview_flutter_android/android/src/main/java/io/flutter/plugins/webviewflutter/WebViewHostApiImpl.java

```diff
@@ -71,27 +71,40 @@ public void setWebContentsDebuggingEnabled(boolean enabled) {
     }
   }
 
-  private static class ReleasableChild<T> {
-    @Nullable private T child;
+  private static class ReleasableValue<T> {
```

Comment:
Why isn't this using bounded types?

Reply:
(Which would avoid the need for the type check in `release`, in addition to making sure incorrect use of ReleasableValue wouldn't compile.)